src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicFormatting.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Microsoft.VisualStudio.IntegrationTest.Utilities.Input;
+using Roslyn.Test.Utilities;
+using Roslyn.VisualStudio.IntegrationTests.Extensions;
+using Roslyn.VisualStudio.IntegrationTests.Extensions.Editor;
+using Xunit;
+
+// *************************************************************
+// ***** Turn on visible whitespace when editing this file *****
+// *************************************************************
+
+namespace Roslyn.VisualStudio.IntegrationTests.VisualBasic
+{
+    [Collection(nameof(SharedIntegrationHostFixture))]
+    public class BasicFormatting : AbstractEditorTest
+    {
+        protected override string LanguageName => LanguageNames.VisualBasic;
+
+        public BasicFormatting(VisualStudioInstanceFactory instanceFactory)
+            : base(instanceFactory, nameof(BasicFormatting))
+        {
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        public void VerifyFormattingIndent()
+        {
+            SetUpEditor(@"
+$$Module A
+    Sub Main(args As String())
+    
```

Comment:
💡 This is a bit risky. It's a good idea to avoid semantically relevant trailing whitespace because some editors can remove it without warning the user. This goes doubly in cases like this where removing the whitespace can only cause false negatives in the test. For example, unlike all the other tests in StyleCop Analyzers, when it came to testing SA1028 we used [StringBuilder.AppendLine](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/StyleCop.Analyzers/StyleCop.Analyzers.Test/SpacingRules/SA1028UnitTests.cs) instead.